Gemfile.lock

```diff
@@ -38,24 +38,24 @@ GEM
       minitest (~> 5.1)
       thread_safe (~> 0.3, >= 0.3.4)
       tzinfo (~> 1.1)
-    addressable (2.5.0)
+    addressable (2.5.1)
```

Comment:
Please don't update the bundle more than is necessary - just do `bundle update social-share-button` rather than updating the whole thing.